myning/objects/player.py

```diff
@@ -118,16 +118,12 @@ def revive_ally(self, ally: Character):
         self.remove_fallen_ally(ally)
         self._allies.append(ally)
 
-    def add_available_exp(self, exp):
+    def add_exp(self, exp):
         if len(self.army) > 1:
             self.exp_available += int(exp * 1 / 2 * len(self.army) * self.macguffin.exp_boost)
         else:
             self.add_experience(int(exp * self.macguffin.exp_boost))
 
-    def add_exp(self, exp):
```

Comment:
I still don't understand why we wouldn't use this anymore? The other one is specifically for applying XP after a trip, this is just to add to the total.

Reply:
In my attempt to simplify things, I opted to just increment `player.exp_available` in the barracks `buy_exp` function. This is not used anywhere else. Do you think it would be better to keep this function instead?

Reply:
I haven't been consistent, but one ideology I'd like to stick to is not manipulating class data directly. I like the idea of only using methods.